extension/js/common/browser/browser-msg.ts

```diff
@@ -429,10 +429,19 @@ export class BrowserMsg {
     BrowserMsg.HANDLERS_REGISTERED_BACKGROUND[name] = handler;
   }
 
-  public static createIntervalAlarm(action: string, periodInMinutes: number) {
+  public static async clearAllDeleteExpiredAlarms() {
+    const alarms = await chrome.alarms.getAll();
+    for (const alarm of alarms) {
+      if (alarm.name.includes('delete_expired_interval_')) {
+        await chrome.alarms.clear(alarm.name);
+      }
+    }
+  }
+
+  public static async createIntervalAlarm(action: string, periodInMinutes: number) {
     const alarmName = `${action}_interval_${Date.now()}`;
 
-    void chrome.alarms.create(alarmName, { periodInMinutes });
+    await chrome.alarms.create(alarmName, { periodInMinutes });
```

Comment:
`chrome.alarms.create` documentation says `If there is another alarm with the same name (or no name if none is specified), it will be cancelled and replaced by this alarm.`.

What about using `${action}_interval` as name (without timestamp), so existing alarm will be automatically cancelled and replaced with new one?